src/lib/frontend/models.ml

```diff
@@ -477,55 +484,63 @@ let rec pp_value ppk ppf = function
 let pp_constant ppf (_sy, t) =
   Fmt.pf ppf "%a" SmtlibCounterExample.pp_abstract_value_of_type t
 
-let output_concrete_model fmt props ~functions ~constants ~arrays =
-  if ModelMap.(is_suspicious functions || is_suspicious constants
-               || is_suspicious arrays) then
-    Format.fprintf fmt "; This model is a best-effort. It includes symbols
-        for which model generation is known to be incomplete. @.";
-
-  Format.fprintf fmt "@[<v 2>(";
-  if Options.get_model_type_constraints () then
-    begin
-      Why3CounterExample.output_constraints fmt props;
-      Format.fprintf fmt "@ ; values"
+let output_concrete_model fmt m =
+  if Options.get_interpretation () then begin
```

Comment:
We should remove this test. Deciding printing models shouldn't be taken in the pretty printer but in some caller of this function.

Reply:
Yes. Created an issue.